examples/webgl_postprocessing_lut.html

```diff
@@ -0,0 +1,289 @@
+const identityLUT = new Uint8Array([
+	   0,   0,   0, 255,   255,   0,   0, 255,    0,   0, 255, 255,   255,   0, 255, 255,
+	   0, 255,   0, 255,   255, 255,   0, 255,    0, 255, 255, 255,   255, 255, 255, 255,
+]);
+
+const lutNameIndexMap = {};
+lutTextures.forEach( function( info, ndx ) {
+	lutNameIndexMap[ info.name ] = ndx;
+	const texture = new THREE.DataTexture( identityLUT, 4, 2 );
+	texture.minFilter = THREE.LinearFilter;
+	texture.magFilter = THREE.LinearFilter;
+	texture.needsUpdate = true;
+	texture.flipY = false;
+
+	info.texture = texture;
+	const lutSize = info.size;
+
+	// set the size to 2 (the identity size). We'll restore it when the
+	// image has loaded. This way the code using the lut doesn't have to
+    // care if the image has loaded or not
+	info.size = 2;
+
+	imgLoader.load( info.url, function( image ) {
+		const width = lutSize * lutSize;
+		const height = lutSize;
+		info.size = lutSize;
+		ctx.canvas.width = width;
+		ctx.canvas.height = height;
+		ctx.drawImage( image, 0, 0 );
+		const imageData = ctx.getImageData( 0, 0, width, height );
+
+		texture.image.data = new Uint8Array(imageData.data.buffer);
+		texture.image.width = width;
+		texture.image.height = height;
+		texture.needsUpdate = true;
+	});
+});
+
+const lutSettings = {
+  lut: lutNameIndexMap.thermal,
+};
+gui.add(lutSettings, 'lut', lutNameIndexMap );
+
+const cameraOrtho = new THREE.OrthographicCamera( -0.5, 0.5, 0.5, -0.5, -1, 1 );
+
+const cameraPerspective = new THREE.PerspectiveCamera( 50, 2, 1, 10000 );
+cameraPerspective.position.z = 900;
+
+const sceneModel = new THREE.Scene();
+const sceneBG = new THREE.Scene();
+
+const directionalLight = new THREE.DirectionalLight( 0xffffff );
+directionalLight.position.set( 0, - 0.1, 1 ).normalize();
+sceneModel.add( directionalLight );
+
+const loader = new THREE.GLTFLoader();
+loader.load( 'models/gltf/LeePerrySmith/LeePerrySmith.glb', function ( gltf ) {
+
+	createMesh( gltf.scene.children[ 0 ].geometry, sceneModel, 100 );
+
+} );
+
+const materialColor = new THREE.MeshBasicMaterial( {
+	map: new THREE.TextureLoader().load( 'textures/cube/SwedishRoyalCastle/pz.jpg' ),
+	depthTest: false,
+} );
+
+quadBG = new THREE.Mesh( new THREE.PlaneBufferGeometry( 1, 1 ), materialColor );
+sceneBG.add( quadBG );
+
+const effectLUT = new THREE.ShaderPass( THREE.LUTShader );
+effectLUT.renderToScreen = true;
+
+const renderBackground = new THREE.RenderPass( sceneBG, cameraOrtho );
+const renderModel = new THREE.RenderPass( sceneModel, cameraPerspective );
+
+renderModel.clear = false;
+
+const rtParameters = {
+	minFilter: THREE.LinearFilter,
+	magFilter: THREE.LinearFilter,
+	format: THREE.RGBFormat,
+};
+composer1 = new THREE.EffectComposer( renderer, new THREE.WebGLRenderTarget( 1, 1, rtParameters ) );
+
+composer1.addPass( renderBackground );
+composer1.addPass( renderModel );
+composer1.addPass( effectLUT )
+
+function resizeRendererToDisplaySize( renderer ) {
```

Comment:
I'm not sure the way `resizeRendererToDisplaySize()` works is okay. Why not just using a `resize` event listener like in the other examples?

Reply:
Like I said in the other comment, it's a matter of consistency. I would prefer a solution where you fix the issue in all examples. Not just in one.